README.md

```diff
@@ -121,9 +123,21 @@ The data flow is as follows:
 3. Server takes post, and passes it to Gravity Forms as POST request
 4. Gravity Forms gets data
 
-Point 3 can be managed in multiple ways, depending on your build.
+Point 3 can be managed in multiple ways, depending on your build. For added security make sure you follow the steps below to implement reCAPTCHA on your form, preventing bots and spam.
```

Comment:
Not relevant to the process of passing data to the server. Try to keep the points as simple and concise as possible.